package.json

```diff
@@ -40,61 +40,56 @@
   ],
   "dependencies": {
     "@launchpadlab/lp-hoc": "^5.0.4",
-    "classnames": "^2.2.5",
-    "cleave.js": "^1.5.3",
-    "filter-invalid-dom-props": "^2.0.0",
-    "lodash": "^4.17.4",
-    "moment": "^2.19.3",
-    "prop-types": "^15.5.8",
-    "react-color": "^2.14.0",
+    "classnames": "^2.3.1",
+    "cleave.js": "^1.6.0",
+    "filter-invalid-dom-props": "^2.1.0",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.2",
+    "prop-types": "^15.8.1",
+    "react-color": "^2.19.3",
     "react-datepicker": "^1.0.3",
-    "react-modal": "^3.11.2",
+    "react-modal": "^3.14.4",
     "react-router": "^3.2.1",
-    "react-switch": "^5.0.1",
-    "redux-flash": "^2.0.1"
+    "react-switch": "^6.0.0",
+    "redux-flash": "^2.0.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.4.0",
-    "@babel/polyfill": "^7.4.3",
-    "@launchpadlab/babel-preset": "^2.1.0",
-    "@launchpadlab/eslint-config": "^2.4.1",
-    "@size-limit/preset-small-lib": "^3.0.0",
-    "@storybook/addon-a11y": "^5.2.5",
-    "@storybook/addon-actions": "^5.0.6",
-    "@storybook/addon-knobs": "^5.2.5",
-    "@storybook/addon-links": "^5.0.6",
-    "@storybook/addon-storysource": "^5.2.5",
-    "@storybook/addon-viewport": "^5.2.5",
-    "@storybook/addons": "^5.0.6",
-    "@storybook/react": "^5.0.6",
-    "babel-loader": "^8.0.0",
-    "bourbon": "^4.3.3",
-    "bourbon-neat": "^1.8.0",
-    "css-loader": "^3.0.0",
-    "documentation": "^12.1.1",
+    "@babel/cli": "^7.17.6",
+    "@babel/core": "^7.17.9",
+    "@launchpadlab/babel-preset": "^2.1.1",
+    "@launchpadlab/eslint-config": "^2.7.0",
+    "@size-limit/preset-small-lib": "^7.0.8",
+    "@storybook/addon-a11y": "^6.4.22",
+    "@storybook/addon-actions": "^6.4.22",
+    "@storybook/addon-links": "^6.4.22",
+    "@storybook/addon-storysource": "^6.4.22",
+    "@storybook/addon-viewport": "^6.4.22",
+    "@storybook/addons": "^6.4.22",
+    "@storybook/react": "^6.4.22",
+    "babel-loader": "^8.2.4",
+    "bourbon": "^7.2.0",
+    "bourbon-neat": "^4.0.0",
+    "core-js": "^3.21.1",
+    "css-loader": "5.2.7",
+    "documentation": "^13.2.5",
     "enzyme": "^3.2.0",
     "enzyme-adapter-react-16": "^1.1.0",
-    "eslint": "^6.2.1",
-    "husky": "^4.2.1",
+    "eslint": "^8.12.0",
```

Comment:
I have a vague memory of eslint v8 not working when you tried to update the ionic template (https://github.com/LaunchPadLab/ionic-client-template/pull/19)?

That might be what's causing this build to fail. Could you please investigate? 
![Image 2022-04-18 at 9 52 35 PM](https://user-images.githubusercontent.com/20426095/163904704-4d9eac73-003a-434a-a5b4-27fe72d729ef.jpg)

Reply:
God, I can't believe you remembered this! I did not. Anyway, you're (I was??) right: our ESLint settings didn't work with the latest version of `eslint`. Downgraded to 7.32.0, which seems to work when `yarn lint` is invoked.